spec/augeas_tree_spec.rb

```diff
@@ -87,6 +87,21 @@
     end
   end
 
+  describe "#select" do
+    it "selects entries according to passed block" do
+      matcher = CFA::Matcher.new(collection: "spec")
+      expect(tree.select(matcher).size).to eq 2
```

Comment:
well, we have matcher, that select all elements in collection and then it verify that it find all 2 elements in collection.